### PR TITLE
add idiom 308 Integer to string in base b

### PR DESCRIPTION
**Pull request by simonvonhackewitz**

### CODE REVIEW

**Comment by simonvonhackewitz** [comment]
Is the output unambiguous? Or would something like "decimal 2 is 10 in base 2" be better?

File: src/idiom308ex.fz [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ex308 is
```
Comment (michaellilltokiwa) [inline src/idiom308ex.fz:1]: We have
```
  # convert this to a number using the given base.  If negative, add "-" as
  # the first character.
  #
  public as_string(base u32) String : character_encodings
```

in stdlib. Did you see this?

Comment (simonvonhackewitz) [reply src/idiom308ex.fz:1]: no 🙈

It doesn't work for 0 in base 999999 but I guess this is rarely needed

Comment (michaellilltokiwa) [reply src/idiom308ex.fz:1]: For base 999999 you would first need 999999 symbols. So no this is useless.

**Review by michaellilltokiwa — APPROVED** [review]

**Pull request merged**